### ISSUE TITLE
Perfect if we add the README.md preview to libtoc

### ISSUE THREAD

**Issue by emerisier** [amochin/robotframework-libtoc#5]
Hi
thanks for this good package, is it possible to add the README.md preview of our project?
Best ragards

**Comment by amochin** [comment]
Hi @emerisier, thanks for the issue!
Could you please elaborate on it, how exactly should a readme.md be embedded in the keywords documentation?

**Comment by emerisier** [comment]
No, just if a link to the README.md (quick access) could be present from your web page (configurable from .libtoc)
But it's true that we go beyond the need for your tool,
Additionally, it would require the document preview, this requires interpreting the markdown code
You can close this request ;)

**Comment by amochin** [comment]
Well, what you can already now, is to customize the template for the homepage - and put a link to the readme file directly there in the HTML file. 
See the explanation here:
https://github.com/amochin/robotframework-libtoc?tab=readme-ov-file#how-to-change-the-toc-and-the-homepage-html-templates

Do you think it solves your request?

**Comment by emerisier** [comment]
thank you, this fits my needs perfectly ;)

Convert md2html
libtoc --output_dir docs --homepage_template README.html .
